packages/turbo/src/VisitableView.tsx

```diff
@@ -143,8 +149,15 @@ const VisitableView = React.forwardRef<RefObject, React.PropsWithRef<Props>>(
       onConfirm
     );
 
+    const {
+      activityIndicator,
+      handleShowActivityIndicator,
+      handleHideActivityIndicator,
+    } = useActivityIndicator(renderLoading);
```

Comment:
I think the naming here could also be improved to not refer to `ActivityIndicator`